### PR TITLE
Updated WallShearProcess py

### PR DESCRIPTION
**Pull request by vickie-ngo**
Implements class from motionProfile.py

### CODE REVIEW

File: MAIN_SCRIPT.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+heaving_amplitude = chord_length*0.5    #[M]
+pitching_amplitude = 70                 #[deg]
+time_steps_per_cycle = 1000             #[-]
+number_of_cycles = 4                    #[-]
+
+
+
+
+
+import motionProfile as mP
```
Comment (anthony-walker) [inline MAIN_SCRIPT.py:23]: Typically, you want to place all imports at the start of the file.

File: ansysFiles/talkToAnsys.py [hunk 2]
```diff
@@ -2,14 +2,15 @@
 import subprocess
 import shutil
 
+fluent_path = shutil.which("fluent")
+
+WB_path=fluent_path[0:int(fluent_path.find("fluent"))] + r"Framework\bin\Win64\RunWB2.exe"
```
Comment (anthony-walker) [inline ansysFiles/talkToAnsys.py:7]: This only works on Windows, you may consider adding content to work with windows or mac/linux

**Review by anthony-walker — APPROVED** [review]
Looks pretty good, I would consider the changes I mentioned.

File: ansysFiles/talkToAnsys.py (hunk 2 above)
Comment (vickie-ngo) [reply ansysFiles/talkToAnsys.py:7]: Do you recommendations on how to make this Mac/Linux compatible?

**Pull request merged**